src/peergos/server/cli/CLI.java

```diff
@@ -45,7 +45,7 @@
         get_follow_requests("Show the users that have sent you a follow request"),
         follow("Send a follow-request to another user.", "follow username-to-follow"),
         passwd("Update your password"),
-        share("","");
+        share_read("share_read path <user>","Grant read access for a file to another user.");
```

Comment:
nit: file or folder

Reply:
Currently it is just files - will  update to include  folders in different PR.

Reply:
Aren't folders exactly the same api? i.e. zero extra work?

Reply:
Yes but at the moment everything in the CLI works for files and not folders which is a nice consistency to have.  When we update get/put etc. for folders we can remove do the same here.